test/theta_sketch_dup_test.cc

```diff
@@ -0,0 +1,95 @@
+#include "theta_sketch_dup.h"
+#include <gtest/gtest.h>
+#include <random>
+#include <set>
+#include <string>
+#include "gen_string.h"
+
+namespace datasketches {
+
+TEST(ThetaSketchDup, TestSerializeDeterminedCaseWithoutEstimation) {
+  // @a: theta sketch with storage set to be [2^5, 2^5*threshold)
+  // @b: theta sketch deserialized from serialized a 
+  // this test data stream only has 20 distinct elements, so the result is exact
+  // instead to estimated
+  auto a = update_theta_sketch_dup::builder().set_lg_k(5).build();
+  for (int i = 0; i < 20; i++) a.update(i);
+  
+  // serialize using proto
+  // a and b should be the same before/after the serialization
+  datasketches_pb::Update_theta_sketch_dup serialized_proto;
+  a.serialize(&serialized_proto);
+  auto b = update_theta_sketch_dup::deserialize(serialized_proto);
+  ASSERT_EQ(a, b);
```

Comment:
These should all be EXPECT calls instead of ASSERTs, as the failure of one doesn't necessarily mean the rest of the test is invalid.

Ditto for the test below.

Reply:
I see, I thought this have to be ASSERT as it must be hold.